scripts/gen-edpm-kustomize.sh

```diff
@@ -199,10 +199,10 @@ cat <<EOF >>kustomization.yaml
       path: /spec/nodes/edpm-compute-${INDEX}/ansibleHost
       value: 192.168.122.$((100+${INDEX}))
     - op: replace
-      path: /spec/nodes/edpm-compute-${INDEX}/hostName
+      path: /spec/roless/edpm-compute-${INDEX}/hostName
```

Comment:
there's an extra trailing `s` in `roles`

Reply:
oops